docs/sources/v0.47.x/using-k6-browser/metrics.md

```diff
@@ -152,7 +152,7 @@ export default async function () {
       JSON.parse(JSON.stringify(window.performance.getEntriesByName('total-action-time')))[0].duration
     );
 
-    myTrend.add(total_action_time);
+    myTrend.add(totalActionTime);
```

Comment:
```suggestion
    myTrend.add(total_action_time);
```

Reply:
Same as other comment. Changed intentionally. Wont work using the named value for the trend here, has to be the variable that has an actual value.